clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached

```diff
@@ -25,4 +25,6 @@ sed -e 's/^-l .*$/-l '$listen_address'/g'\
     -e 's/^# *-v *$/-v/g'\
     -e 's/^\(# *\|\)-c.*$/-c 4096/g' </etc/memcached.conf >/etc/memcached_11211.conf
 
+echo "\n# Disable listening on UDP\n-U 0" >> /etc/memcached_11211.conf
+
```

Comment:
It looks like it's the other way round - this works if the shebang is /bin/sh (meaning we're using the dash shell) but doesn't if it's /bin/bash. Ugh. I agree with you @GAB-MS that we shouldn't be relying on this difference and should be using printf instead.